### ISSUE TITLE
Assets be set to On, Display layers set to have R flag

### ISSUE THREAD

**Issue by YasinH** [Kif11/lanoria-assembly#23]
Changing the Geo type on all assets to On instead of Reference and then set the R flag on the display layers. This way, whenever the lighter wants to select an object from viewport, it's as easy as turning off the R flag from display layer without having to go through all the assets from outliner to find that particular asset.

**Comment by Kif11** [comment]
@YasinH Some update on that. Most of the assets has Drawing Override enable witch compete with the layer override attributes. Ideally it should be uncheck on the *model* file level not in the shot scene which is problematic.

It could be uncheck in every scene when it will be stored as a reference edit which is not cool.

**Comment by YasinH** [comment]
Good catch on this. That's what the Geo type is linked to then. It is not coming from the model, it's part of the rig and autorig script which is something we don't have access to. Breaking connections as a reference edit is not cool indeed. Let's hold this issue off for the time being. Actually, this won't become something to deal with later as we are going to go full-cache mode. In that case, we are only dealing with the models which we have full access to.

**Comment by Kif11** [comment]
All right.
